packages/form-js-editor/test/spec/properties-panel/PropertiesPanel.spec.js

```diff
@@ -213,6 +213,102 @@ describe('properties panel', function() {
     });
 
 
+    describe('select', function() {
+
+      it('entries', function() {
+
+        // given
+        const field = schema.components.find(({ key }) => key === 'language');
+
+        const result = createPropertiesPanel({
+          container,
+          field
+        });
+
+        // then
+        expectGroups(result.container, [
+          'General',
+          'Values',
+          'Validation'
+        ]);
+
+        expectGroupEntries(result.container, 'General', [
+          'Field Label',
+          'Field Description',
+          'Key'
+        ]);
+
+        expectGroupEntries(result.container, 'Values', [
+          [ 'Label', 2],

```

Comment:
Inconsistent spacing.

Reply:
Fixed [here](https://github.com/bpmn-io/form-js/pull/56/commits/f7fdcd065feaf001527448d83fc2245a156b195d#diff-b3692abfd67feb9d0fcd6b6b6308c387e7b4dc5b88296b87bd5f38b1667cb82eR146) and [here](https://github.com/bpmn-io/form-js/pull/56/commits/4f18ffcd2a2850771348a50724e7874b8a258e2a#diff-b3692abfd67feb9d0fcd6b6b6308c387e7b4dc5b88296b87bd5f38b1667cb82eR242).